tests/test_vector2.py

```diff
@@ -377,6 +377,44 @@ def test_angle(self):
         with self.assertRaises(ZeroDivisionError):
             angleResult = v1.angle(Vector2(0, 0))
 
+    def test_polar(self):
+        # polar <-> cartesian conversions
+        cases = [
```

Comment:
This is great! Hinting that `vectormath` could use a better testing framework. (And I totally agree - I'm thinking `@pytest.mark.parametrize(...)`)